cli/README.md

```diff
@@ -94,3 +94,27 @@ Get Server-Timing header medians _and_ all individual run values:
 ```
 wpt-server-timing --test 221011_AiDcV7_GGM --include-runs
 ```
+
+### `benchmark-url`
+
+Sends the selected number of requests with a certain concurrency to provided URLs to find out the median response time for each URL. It can also track Server-Timing metrics and get media values for each of them.
```

Comment:
```suggestion
Sends the selected number of requests with a certain concurrency to provided URLs to find out the median response time for each URL. It can also track Server-Timing metrics and get median values for each of them.
```